wordclock/settings.h

```diff
@@ -72,5 +72,7 @@ void SettingsScheduleStore();
 void SettingsCheckStore();
 void SettingsClockReset();
 
+void SettingRandomColors();
```

Comment:
Dit is niet echt een setting?
kan je het niet beter onder de functie van FilterRainbow zetten?

Reply:
In deze functie worden random kleuren ingesteld. Het is eigenlijk geen filter. Zou beter kunnen maar dan moet de http pagina overhoop. Dit was quick and dirty. En wat mij betreft duidelijk genoeg.

Reply:
Vond vooral de plek van de functie een beetje gek gezien de rest allemaal te maken heeft met de flash settings
maar prima ik had me zelf assigned als reviewer dus merge maar als je er mee klaar bent.